src/plugins/kdb/db2/libdb2/btree/bt_seq.c

```diff
@@ -239,13 +240,17 @@ __bt_seqset(t, ep, key, flags)
 				return (RET_ERROR);
 
 			/* Check for an empty tree. */
-			if (NEXTINDEX(h) == 0) {
+			if (NEXTINDEX(h) <= 0) {
 				mpool_put(t->bt_mp, h, 0);
 				return (RET_SPECIAL);
 			}
 
 			if (h->flags & (P_BLEAF | P_RLEAF))
 				break;
+			if (UNDERFLOW_NEXTINDEX(h) || NEXTINDEX(h) <= 1) {
```

Comment:
Again I think I would like to defer this change, until one of the BSDs introduces something similar.  (I don't have an analysis showing that an overflow is unlikely for this case as I do for cachesize, however.)